conventions.md

```diff
@@ -95,14 +95,15 @@ Each representation of a phoneme is first normalized into a canonical decomposit
 
 The ordering conventions used in PHOIBLE are as follows:
 
-1. **Place features:** linguolabial ~~`̼`~~, dental ~~`̪`~~, apical ~~`̺`~~, laminal ~~`̻`~~, advanced ~~`̟`~~, retracted ~~`̠`~~
-2. **Manner features:** raised ~~`̝`~~, lowered ~~`̞`~~, advanced tongue root ~~`̘`~~, retracted tongue root ~~`̙`~~, frictionalized ~~`͓`~~
-3. **Secondary articulations:** more round ~~`̹`~~, less round ~~`̜`~~, derhoticized ~~`̮`~~
-4. **Laryngeal settings:** creaky ~~`̰`~~, breathy ~~`̤`~~, voiced ~~`̬`~~, devoiced (below) ~~`̥`~~, devoiced (above) ~~`̊`~~
-5. **Syllabicity:** syllabic ~~`̩`~~ and non-syllabic markers ~~`̯`~~
-6. **Vowel quality modifications**: nasalized ~~`̃`~~, centralized ~~`̈`~~, mid-centralized ~~`̽`~~
-7. **Stop release:** unreleased `̚`
-8. **Spacing modifier letters:** rhotic hook `˞`, nasal release `ⁿ`, lateral release `ˡ`, labialized `ʷ`, palatalized `ʲ`, velarized `ˠ`, pharyngealized `ˤ`, glottalized `ˀ`, aspirated `ʰ`, ejective `ʼ`, long `ː`, half long `ˑ`
+1. **Place features:** velarized/pharyngealized ~~`̴`~~, linguolabial ~~`̼`~~, dental ~~`̪`~~, apical ~~`̺`~~, laminal ~~`̻`~~, advanced ~~`̟`~~, retracted ~~`̠`~~
+2. **Manner features:** non-sibilant ~~`͇`~~, raised ~~`̝`~~, lowered ~~`̞`~~, advanced tongue root ~~`̘`~~, retracted tongue root ~~`̙`~~, frictionalized ~~`͓`~~
+3. **Secondary articulations:** more round ~~`̹`~~, less round ~~`̜`~~
+4. **Laryngeal settings:** creaky ~~`̰`~~, breathy ~~`̤`~~, voiced ~~`̬`~~, stiff ~~`̬`~~, devoiced (below) ~~`̥`~~, devoiced (above) ~~`̊`~~, fortis ~~`͈`~~, lenis ~~`͉`~~
```

Comment:
@drammock - we seem to support both above and below devoice markers:

https://github.com/phoible/dev/blob/master/scripts/aggregation-helper-functions.R#L400-L401

We discuss the issue here: #225 -- and I suppose I'm the bottle neck on going in and fixes the sources and doing a PR. I'll bump this up on my list.

Reply:
since we already have a separate issue to track the above/below devoicing ring, I'll consider it not a blocker for this PR.